R/rarefyAssay.R

```diff
@@ -27,10 +27,12 @@
 #'   (Please use \code{assay.type} instead. At some point \code{assay_name}
 #'   will be disabled.)
 #'   
-#' @param min_size A single integer value equal to the number of counts being 
+#' @param sample A single integer value equal to the number of counts being 
 #'   simulated this can equal to lowest number of total counts 
 #'   found in a sample or a user specified number. 
 #'   
```

Comment:
The name "sample" was chose because vegan uses the same naming. However, phyloseq uses min_size...

But one important point is that when rarefied beta diversities are calculated, one should use avgdist() function from vegan (and that has sample parameter)